### ISSUE TITLE
Feature request: write iamge to clipboard

### ISSUE THREAD

**Issue by Yolwoocle** [love2d/love#1766]
Currently, `love.system.setClipboardText` only supports text, and not images. I would appreciate it if we could write images to the clipboard too, in the same way selecting part of an image in MS Paint, then pasting it onto a different program, works.

**Comment by zorggn** [comment]
What about OS cross-compatibility? How would this work on Linux / OSX?

Even Windows has a pretty complicated clipboard system if you looked into it, Linuxes don't even have just one option in regards to the clipboard implementation (and more importantly, the interface).

**Comment by Yolwoocle** [comment]
How does LÖVE handle clipboard on Linux right now? Would implementing this be dissimilar to how setClipboardText currently works?

**Comment by slime73** [comment]
It just [calls](https://github.com/love2d/love/blob/main/src/modules/system/sdl/System.cpp#L63) SDL's `SDL_SetClipboardText` function. SDL does not have a clipboard API for content that isn't text, so unless that gets added to SDL (I don't think there are plans for that at the moment), there's not much love's own code can do.

**Comment by learn-more** [comment]
SDL has implemented it.

Are there any major blockers for the move to SDL3?

**Comment by slime73** [comment]
Yes, the major blocker is that SDL3 is not released yet and won't be for several months at least.

I have a branch that conditionally uses SDL3 when love is compiled to use it, and I hope we can switch to fully using SDL3 everywhere for love 12's release, but that branch needs more testing and I won't make a final love 12.0 release depend on an unreleased SDL version, so we'll have to see how SDL3's release goes.

I also expect the first few SDL 3 releases will have a lot more bugs than SDL2 has right now, since so much has changed.

**Comment by learn-more** [comment]
> Yes, the major blocker is that SDL3 is not released yet and won't be for several months at least.
> 
> I have a branch that conditionally uses SDL3 when love is compiled to use it, and I hope we can switch to fully using SDL3 everywhere for love 12's release, but that branch needs more testing and I won't make a final love 12.0 release depend on an unreleased SDL version, so we'll have to see how SDL3's release goes.
> 
> I also expect the first few SDL 3 releases will have a lot more bugs than SDL2 has right now, since so much has changed.

My bad, I assumed that it was released already.

**Comment by gustavostuff** [comment]
Subscribed to this thread. I am currently working on an NES graphics editors and this being added to SDL/Love2D would be awesome for the workflow.

<img width="644" height="390" alt="Image" src="https://github.com/user-attachments/assets/ad1a3074-3567-4531-90f0-2cf74287a8b5" />
